openmc/cell.py

```diff
@@ -528,11 +529,12 @@ def clone(self, memo=None):
         # If no nemoize'd clone exists, instantiate one
         if self not in memo:
             # Temporarily remove paths
-            paths = self.paths
+            paths = self._paths
             self._paths = None
```

Comment:
Good point -- I think I'll actually just set it to `None` in the constructor to avoid creating an unnecessary list if paths are not being used.